src/modules/core/components/Dialog/DialogSection.tsx

```diff
@@ -5,6 +5,7 @@ import { getMainClasses } from '~utils/css';
 import styles from './DialogSection.css';
 
 interface Appearance {
+  theme?: 'heading' | 'sidePadding';
```

Comment:
@ArmandoGraterol Alicjia created these two new themes for the `DialogSectio`.

Please wait with your #2300 until this PR is merged, then rebase on the changes so you can use these the same way they are used here so that all our modal follow the same design.